glfs.c

```diff
@@ -371,22 +378,34 @@ static glfs_t* tcmu_create_glfs_object(char *config, gluster_server **hosts)
 		goto unref;
 	}
 
+	ret = snprintf(logfilepath, PATH_MAX, "%s/%s",
+	               tcmu_log_dir?tcmu_log_dir:DEFAULT_LOG_DIR, TCMU_GLFS_LOG_FILENAME);
```

Comment:
Instead of having the glfs handler module know this many details about the runner logging code, I think it would be nicer to make s function that handles this.